e2e/common/traits/health_test.go

```diff
@@ -439,7 +437,7 @@ func TestHealthTrait(t *testing.T) {
 					var r *v1.HealthCheckResponse
 
 					for h := range c.Pods[0].Health {
-						if c.Pods[0].Health[h].Name == "camel-routes" && c.Pods[0].Health[h].Status == "DOWN" {
+						if c.Pods[0].Health[h].Name == "never-ready" && c.Pods[0].Health[h].Status == "DOWN" {
```

Comment:
This one look weird. If it is expected the Integration name, then, a randomized name variable `name` should be provided instead. Are we sure this condition is really existing?

Reply:
We are sure because is the condition that we explicitly define in files/NeverReady.java